hdf5.spec

```
@@ -62,7 +62,7 @@ BuildRequires: Modules
 %else
 BuildRequires: environment-modules
 %endif
-Provides:       %{name}-cart-%{cart_major}-daos-%{daos_major}
+Provides: %{name}-cart-%{cart_major}-daos-%{daos_major}
```

We don't really need the cart major in here any more with cart being tied to daos now.  I'm inclined to get rid of it and just assume we won't want to test 1.0 with any newer hdf5 RPM.  And we can always go fix 1.0 if we do.
```suggestion
Provides: %{name}-daos-%{daos_major}
```